signaling-server/room_manager.py

```diff
@@ -75,7 +75,7 @@ async def send_to_all(self, event, data=None, room=None, skip_id=None):
 
             for client in clients:
                 if client['id'] != skip_id:
-                    tasks.append(self._sio.emit(event, data, to=client['id']))
+                    tasks.append(asyncio.create_task(self._sio.emit(event, data, to=client['id'])))
```

Comment:
Implicit conversion from coroutine to task by asyncio.wait deprecated in python3.9